net/port.ts

```diff
@@ -0,0 +1,29 @@
+/** @returns an open port */
+export function getAvailable(): number {
```

Comment:
How about `getAvailablePort`? This belongs to `net` module. I think it's better to clarify what is available in API name.